joomla/joomla-cms
# [4.4] Smart Search: Properly sort filters

Pull Request for Issue #41930.

### Summary of Changes
The entries in the filters in Smart Search aren't properly ordered in the frontend.


### Testing Instructions
1. Create a website with some content, especially nested categories. Best solution would be to use the testing sampledata.
2. Go to the Smart Search search view in the frontend and open the advanced filters
3. Check for example the category filter


### Actual result BEFORE applying this Pull Request
The categories are pretty much random.


### Expected result AFTER applying this Pull Request
The categories are displayed properly like in the tree structure.


### Link to documentations
Please select:
